osxcollector/output_filters/analyze.py

```diff
@@ -51,7 +52,7 @@ class AnalyzeFilter(ChainFilter):
     effect the operations of the next filter.
     """
 
-    def __init__(self, no_opendns=False, no_virustotal=False, no_shadowserver=False, readout=False, **kwargs):
+    def __init__(self, no_opendns=False, no_virustotal=False, no_shadowserver=False, no_alexaranking=False, readout=False, **kwargs):
```

Comment:
Since `alexaranking` is technically two words, I guess the parameter should be spelled `no_alexa_ranking`. Why not calling it simply `no_alexa`? It's shorter ;)

Reply:
Done!